### PR TITLE
doc: adding JAVA Example in Step 4: Closing Sessions

### PR DESCRIPTION
**Pull request by jaemyoun**
Added Java Example in Step 4 for safely closing an instance.

Signed-off-by: Jaemyoun Lee <jaemyoun@gmail.com>

### CODE REVIEW

File: doc/rados/api/librados-intro.rst [hunk 1]
```diff
@@ -967,6 +967,15 @@ C++ Example
 	cluster.shutdown();
 
 
+Java Example
+--------------
+
+.. code-block:: java
+
+	cluster.shutDown();
+	cluster.ioCtxDestroy(io);
```
Comment (tchaikov) [inline doc/rados/api/librados-intro.rst]: better off calling `cluster.ioCtxDestroy(io)` before `cluster.shutDown()`

Comment (jaemyoun) [inline doc/rados/api/librados-intro.rst]: fixed it

**Comment by tchaikov** [comment]
@jaemyoun could you squash these two commits into a single one? then it will be good to merge.

**Comment by jaemyoun** [comment]
@tchaikov I squash the commits.

**Comment by tchaikov** [comment]
could you prefix the title of your commit message with the subcomponent your are changing ? see https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst#preparing-and-sending-patches, in your case, "doc: " would suffice.

**Comment by jaemyoun** [comment]
@tchaikov I changed the commit message. is it right?

**Comment by tchaikov** [comment]
right!

**Pull request merged**